### PR TITLE
Avoid debconf warnings

### PR DESCRIPTION
**Pull request by vijayreddy1991**
https://github.com/Shippable/pm/issues/10795

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,8 @@
 FROM ubuntu:14.04
 
+ENV HOME /root
```
Comment (ric03uec) [inline Dockerfile]: why is this needed. we're already exporting this in the `install.sh` file

File: Dockerfile [hunk 2]
```diff
@@ -1,5 +1,7 @@
 FROM ubuntu:14.04
 
+ENV DEBIAN_FRONTEND noninteractive
```
Comment (ric03uec) [inline Dockerfile]: this line is already there in install.sh. why is this required here?

Comment (vijayreddy1991) [reply Dockerfile]: Yes, the above line is in install.sh but i am getting same warnings. So i put in docker file and i checked locally  its working
